includes/class-wc-admin-api-init.php

```diff
@@ -445,6 +508,26 @@ private static function get_schema() {
 				PRIMARY KEY (action_id),
 				KEY note_id (note_id)
 				) $collate;
+			CREATE TABLE {$wpdb->prefix}wc_customer_lookup (
+				customer_id BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+				user_id BIGINT UNSIGNED DEFAULT NULL,
+				username varchar(60) DEFAULT '' NOT NULL,
+				first_name varchar(255) NOT NULL,
+				last_name varchar(255) NOT NULL,
+				email varchar(100) NOT NULL,
+				orders_count BIGINT UNSIGNED NOT NULL,
+				total_spend double DEFAULT 0 NOT NULL,
+				avg_order_value double DEFAULT 0 NOT NULL,
+				date_last_active timestamp DEFAULT '0000-00-00 00:00:00' NOT NULL,
+				date_last_order timestamp DEFAULT '0000-00-00 00:00:00' NOT NULL,
+				date_registered timestamp DEFAULT '0000-00-00 00:00:00' NOT NULL,
```

Comment:
@timmyc

I was just following the pattern of the other report tables here - why are we using `timestamp` instead of `datetime`?

Also, I think I'm going to allow these columns to be null. They won't always have a value, and the all-zero default is causing stuff like `-0001-11-30T00:00:00` when processed through `wc_rest_prepare_date_response()`.

Reply:
> I was just following the pattern of the other report tables here - why are we using timestamp instead of datetime

I'm not entirely sure the background on that decision, but am guessing it might be detailed in some old posts on +protonpower - but perhaps @claudiosanches  or @peterfabian might know why.

Reply:
Allowed the `date_registered` column to be null in 91c8495.